ws-base.coffee

```diff
@@ -24,11 +24,20 @@ class AlienWsBase extends AlienCommander
     @_addWebSocket ws if ws?
     return @
 
-  send: (data, options = {}) ->
-    if (error = @_checkSend data, options)?
+  send: (data, flags, cb) ->
+    if !cb? and _.isFunction flags
```

Comment:
Ezt a három sort ki lehetne faktorolni innen, meg a sendBinary-ból. Meg talán a ws-jsonban is van ugyanez a pattern.